🌐 Update Portuguese Translation for `docs/pt/docs/deployment/https.md`

Joao-Pedro-P-Holanda opened the pull request.

Joao-Pedro-P-Holanda committed:
🌐 Update Portuguese Translation for `docs/pt/docs/deployment/https.md`

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

ceb10n commented on docs/pt/docs/deployment/https.md:
```diff
@@ -14,38 +14,187 @@ Para aprender o básico de HTTPS de uma perspectiva do usuário, verifique <a hr
+Mas uma vez que você saiba o básico de **HTTPS para desenvolvedores**, você pode combinar e configurar diferentes ferramentas facilmente para gerenciar tudo de uma forma simples.
+
+Em alguns dos próximos capítulos, eu mostrarei para você vários exemplos concretos de como configurar o **HTTPS** para aplicações **FastAPI**.
+🔒
```
I think you accidentally hit enter here :D

![emoji](https://github.com/user-attachments/assets/db42ba39-81cf-4463-9c21-5436f09add48)

ceb10n reviewed (approved):
Really great @Joao-Pedro-P-Holanda 😎 

Just one silly suggestion :D

Joao-Pedro-P-Holanda committed:
chore: addressing @ceb10n first review

adding article before Let's encrypt on line 52

moving emoji from line 200 to 199

ceb10n reviewed (approved):
🚀

alejsdev committed:
Merge branch 'master' into docs/update-https

alejsdev commented:
Thanks for the update @Joao-Pedro-P-Holanda :nerd_face: 
And thanks for the review @ceb10n :rocket:
